docs/INSTALL.md

```diff
@@ -0,0 +1,51 @@
+# Install Stargz Snapshotter
+
+We assume that you are using containerd (> v1.4.2) as a CRI runtime.
+
+## Install Stargz Snapshotter with Systemd
+
+- Download release tarfile from [the release page](https://github.com/containerd/stargz-snapshotter/releases). For example, amd64 binary of v0.5.0 is available from https://github.com/containerd/stargz-snapshotter/releases/download/v0.5.0/stargz-snapshotter-v0.5.0-linux-amd64.tar.gz.
+
+- Add the following configuration to containerd's configuration file (typically: /etc/containerd/config.toml). Please see also [an example configuration file](https://github.com/containerd/stargz-snapshotter/blob/master/script/config/etc/containerd/config.toml).
```

Comment:
The link should be `../script/config/etc/containerd/config.toml`

Reply:
I have fix it